wallet/src/ui/app/redux/slices/account/index.ts

```diff
@@ -66,6 +79,38 @@ const accountSlice = createSlice({
             }),
 });
 
-export const { setMnemonic } = accountSlice.actions;
+export const { setMnemonic, setAddress } = accountSlice.actions;
 
 export default accountSlice.reducer;
+
+export const accountCoinsSelector = createSelector(
```

Comment:
See comment in wallet/src/ui/app/redux/slices/sui-objects/index.ts, you should be able to delete a lot of code in this file by leveraging https://github.com/MystenLabs/fastnft/blob/16c817330275a97fac67534a104f8a363571efea/sdk/typescript/src/types/framework.ts#L14